Plugins/BuildServerIntegration/TeamCityIntegration/TeamCityAdapter.cs

```diff
@@ -568,7 +575,8 @@ public void Dispose()
             _httpClient?.Dispose();
         }
 
-        public Project? GetProjectsTree()
+ #nullable disable
+        public Project GetProjectsTree()
```

Comment:
This can return null. Keep the annotation? Use `#nullable disable warnings` instead?